### DISCUSSION TITLE
PCAP file analyze PROBLEM !

### DISCUSSION THREAD

**Discussion by th3ba1r4m** [Security-Onion-Solutions/securityonion#5206] [Unsupported Versions]
Why I cant take  every PCAP file on my logs ?   I cant see my last days pcap files or sometimes recently days files is slowly loading or not showing anything.

![E8_bWMiWQAI3OoD](https://user-images.githubusercontent.com/87234155/130071947-30b2664a-af17-42da-810f-cfaabf97ba44.png)

**Comment by weslambert** [comment]
Is there data inside of `/nsm/pcap` on the sensor? Depending on your disk space, or if traffic is being filtered out, you may not have relevant PCAP on disk.

**Reply by th3ba1r4m** [reply]
inside of it is have to anything or not ? My disk space is nearly  1TB.How many will better for disk space ? what means filtered out ? 

Thanks on advance
